giant_molecular_clouds_tutorial.ipynb

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
<p>WHy is this not complete?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Go away, kindly.

Reply:
Is this now resolved?